src/rgw/rgw_lc.cc

```diff
@@ -1422,6 +1422,16 @@ std::string rgwlc_s3_expiration_header(
       return hdr;
   } /* catch */
 
+  /* dump tags at debug level 16 */
+  RGWObjTags::tag_map_t obj_tag_map = obj_tagset.get_tags();
+  if (cct->_conf->subsys.should_gather(ceph_subsys_rgw, 16)) {
```

Comment:
@mattbenjamin argh. Not a comment about this specific use, but in general (a slight digression) ... we really should have some syntactic sugar for these checks. In most cases the subsys is defined, So having something like: `if (check_ldout(cct, 16))` is probably trivial.

Reply:
neither is great, and moreover, we have an open issue apparently with performance of new version of options